### ISSUE TITLE
Docker logging driver

### ISSUE THREAD

**Issue by foolioo** [rancher-sandbox/rancher-desktop#1874]
### Actual Behavior

Docker has a builtin [support for multiple logging drivers](https://docs.docker.com/config/containers/logging/configure/#supported-logging-drivers), which seems not supported when using Rancher Desktop.

I am running Rancher Desktop version 1.1.1 on MacOS 12.3 with dockerd and K8s disabled. Is it supported or is it a bug?
Relates to #1484.

### Steps to Reproduce

Given a Docker Compose file
```shell
$ docker-compose config
services:
  fluentd:
    container_name: fluend
    image: fluent/fluentd:v1.14.4-1.0
    networks:
      default: null
  nginx:
    container_name: nginx
    depends_on:
      fluentd:
        condition: service_started
    image: nginx:latest
    logging:
      driver: fluentd
      options:
        fluentd-address: fluentd:24224
        tag: nginx.access
    networks:
      default: null
networks:
  default:
    name: example_default
```

When spinning up an Nginx container, an error pops
```shell
$ docker-compose up -d nginx
[+] Running 2/3
 ⠿ Network example_default  Created                                                                                                   0.0s
 ⠿ Container fluend       Started                                                                                                   0.6s
 ⠿ Container nginx        Starting                                                                                                  1.3s
Error response from daemon: failed to initialize logging driver: dial tcp: lookup fluentd on 192.168.5.3:53: no such host
```

it seems that Fluentd is resolved to [Lima DNS](https://github.com/lima-vm/lima/blob/master/docs/network.md#dns-19216853).

### Result

Error response from daemon: failed to initialize logging driver: dial tcp: lookup fluentd on 192.168.5.3:53: no such host

### Expected Behavior

No errors and container logs will be shipped to FluentD

### Additional Information

The lack of Docker logging drivers support, makes the transition from Docker Desktop to Rancher Desktop problematic when utilizing the logging drivers.

### Rancher Desktop Version

1.1.1

### Rancher Desktop K8s Version

1.23.3

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

MacOS Monterey 12.3

### What CPU architecture are you using?

x64

**Comment by foolioo** [comment]
@ericpromislow any update?

**Comment by ericpromislow** [comment]
No, the issue hasn't been scheduled yet. You can see our current project plan at https://github.com/rancher-sandbox/rancher-desktop/projects/1

**Comment by jandubois** [comment]
DNS handling inside the VM has been significantly reworked since v1.1.1. Version 1.2.0 included DNS improvements on all platforms, and further DNS fixes shipped in v1.6.0 and v1.17.1. The Lima DNS address (192.168.5.3) referenced in the error is no longer the resolver used in the current architecture. If Docker containers still cannot resolve service names when using the fluentd logging driver on v1.22.0, please open a new issue with updated reproduction steps.
